### PR TITLE
DDF-2824 filled in missing table information for GeoWebCache docs

### PR DESCRIPTION
**Pull request by ricklarsen**
#### What does this PR do?

Fills in missing values for the `Adding GeoWebCache layers` table in the documentation.

#### Who is reviewing it? 
@bakejeyner @austinsteffes @ahoffer 

#### Select relevant component teams: 
@codice/docs 
#### Ask 2 committers to review/merge the PR and tag them here.
@brjeter
@clockard
@garrettfreibott
@lessarderic
@mcalcote
@ricklarsen - Documentation
@shaundmorris

#### How should this be tested?
verify content additions

#### What are the relevant tickets?
[DDF-2824](https://codice.atlassian.net/browse/DDF-2824)

#### Checklist:
- [x] Documentation Updated
- [ ] Update / Add Threat Dragon models
- [ ] Update / Add Unit Tests
- [ ] Update / Add Integration Tests

#### Notes on Review Process
Please see [Notes on Review Process](https://codice.atlassian.net/wiki/spaces/DDF/pages/71946981/Pull+Request+Guidelines) for further guidance on requirements for merging and abbreviated reviews. 

#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.

### CODE REVIEW

**Comment by ricklarsen** [comment]
build now

**Comment by ricklarsen** [comment]
build now

**Comment by ricklarsen** [comment]
rebased to build now

File: distribution/docs/src/main/resources/content/_appReferences/mg-geowebcache.adoc [hunk 1]
```diff
@@ -48,42 +48,37 @@ Add layers to the local cache:
...
 .Add Layer
-[cols="1,1m,1,2,1,1" options="header"]
+[cols="1,1m,1,2,1" options="header"]
 |===
 |Name
 |Property
 |Type
 |Description
 |Default Value
-|Required
```
Comment (bakejeyner) [inline distribution/docs/src/main/resources/content/_appReferences/mg-geowebcache.adoc]: ❓ What is the motivation for taking out the `Required` table entry?

Comment (ricklarsen) [reply distribution/docs/src/main/resources/content/_appReferences/mg-geowebcache.adoc]: I don't think that it provides any useful information for the person doing the configuring.

**Pull request merged**